README.md

```diff
@@ -1,32 +1,504 @@
+
+  ```
+  <key>NSAppTransportSecurity</key>
+  <dict>
+   <key>NSAllowsArbitraryLoads</key>
+   <true/>
+  </dict>
+  ```
+
+  This is needed, as even HTTPS YouTube links use non-secure endpoints that keep changing over time.
+
+2. By default, Swrve stores all customer data and content in our US data center. If you require EU-only data storage, you must configure the SDK to point to Swrve’s EU-based URL endpoints.
+
+  Add one of the following code snippets to the didFinishLaunchingWithOptions method of your AppDelegate, depending on whether your data will be stored in our US or EU data center:
+
+  US Data Storage
+
+  ```
+  #import "swrve.h"
+  - (BOOL) application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+     [Swrve sharedInstanceWithAppID:<app_id> apiKey:@"<api_key>" launchOptions:launchOptions];
+  }
+  ```
+
+  EU Data Storage
+
+  ```
+  #import "swrve.h"
+  - (BOOL) application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+     SwrveConfig* config = [[SwrveConfig alloc] init];
+     config.selectedStack = SWRVE_STACK_EU;
+     [Swrve sharedInstanceWithAppID:<app_id> apiKey:@"<api_key>" config:config launchOptions:launchOptions];
+  }
+  ```
+
+  For more information, see [How Do I Configure the Swrve SDK for EU Data Storage?](http://docs.swrve.com/faqs/sdk-integration/configure-sdk-for-eu-data-storage/) If you have any questions or need assistance configuring the SDK for EU data storage, please contact support@swrve.com.
+
+3. Replace <app_id> and <api_key> with your Swrve app ID and Swrve API key.
+
+4. (Optional): If you are planning to use or ask for location tracking permission, add the following keys to your `<App>-Info.plist` if not already there (edit the Value message to whatever text you want):
+
+  * `NSLocationAlwaysUsageDescription`. Value: “Location is needed to offer all functionality in this app”.
+  * `NSLocationWhenInUseUsageDescription`. Value: “Location is needed to offer all functionality in this app”.
+
+In-app Messaging
+-
+Integrate the in-app messaging functionality so you can use Swrve to send personalized messages to your app users while they’re using your app. If you’d like to find out more about in-app messaging, see [Intro to In-App Messages](http://docs.swrve.com/user-documentation/in-app-messaging/intro-to-in-app-messages/).
+
+Before you can test the in-app message feature in your game, you need to create an In App Message campaign in the Swrve Dashboard.
+
+For iOS, if you want to display an in-app message as soon as your view is ready, you must trigger the display event when the view controller is in the `viewDidAppear` state and not in `viewDidLoad` state.
```

Comment:
This seems odd here, since events and triggers haven't really been mentioned before now.  It could maybe be added later in a 'use-cases' section?